### PR TITLE
update to latest upstream update-component-sha, which is better

### PR DESCRIPTION
**Pull request by deitch**
And since we reference the explicit hash, no worry about it changing or going away on us. It is pinned.

cc @rvs

### CODE REVIEW

File: tools/update-component-sha.sh [hunk 1]
```diff
@@ -1,71 +1,5 @@
 #!/bin/sh
 set -e
 
-##
-#
-# script to replace hashes in config files
-# see usage() for usage and functionality
-#
+curl -L https://raw.githubusercontent.com/linuxkit/linuxkit/46ef83c97931753c27f1ee4f5616d437d0d8f2d6/scripts/update-component-sha.sh | sh -s - "$@"
```
Comment (rvs) [inline tools/update-component-sha.sh:4]: No, no -- that's not good. I really don't like this style of `curl | sh` at all (for obvious reasons).

**Review by rvs — CHANGES_REQUESTED** [review]
Let's just import it

File: tools/update-component-sha.sh (hunk 1 above)
Comment (deitch) [reply tools/update-component-sha.sh:4]: You are causing trouble! Well, as long as I am not the one who has to keep it up to date...

**Comment by deitch** [comment]
Imported, as you requested. I am sure yetus will yell bloody murder because, well, yetus.

**Comment by deitch** [comment]
Fixed yetus complaints.

**Comment by deitch** [comment]
That took way too many runs, but yetus _finally_ is green.

@rvs independent of what we do with #2353, we should merge this one in.

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**